lib/kdump_utils.pm

```diff
@@ -83,16 +83,21 @@ sub prepare_for_kdump {
 
     # disable packagekitd
     quit_packagekit;
+    my @pkgs = qw(yast2-kdump kdump);
     if ($test_type eq 'before') {
         # On ppc64le, sometime the console font will be distorted into pseudo graphics characters.
         # we need to reset the console font.
         assert_script_run('/usr/lib/systemd/systemd-vconsole-setup') if check_var('ARCH', 'ppc64le');
-        zypper_call('in yast2-kdump kdump');
     }
     else {
-        zypper_call('in yast2-kdump kdump crash');
+        push @pkgs, qw(crash);
     }
-    zypper_call('in mokutil') if is_jeos && get_var('UEFI') && !check_var('ARCH', 'aarch64');
+
+    if (is_jeos && get_var('UEFI')) {
+        push @pkgs, is_aarch64 ? qw(mokutil shim) : qw(mokutil);
```

Comment:
I haven't found a specific subroutine that would determine between `rpi` and `aarch64` as `shim` should be included every except of rpi (https://build.suse.de/request/show/239027)

Reply:
Installing a pkg that is already installed doesn't break anything so that doesn't really matter anyway.

Reply:
Indeed, but as it should come pre-installed, this way we can possibly catch an unintentional drop of it in the kiwi template.